go.mod

```diff
@@ -7,12 +7,3 @@ require (
 	github.com/gofiber/template v1.7.1
 	github.com/mattn/go-sqlite3 v1.14.15
 )
-
-require (
```

Comment:
@wliao008 can you remove the changes from `go.mod`

Reply:
hi @Shivansh-yadav13 those indirect refs were actually removed by the go build system, don't think they're needed as each respective module has their own go.mod? if you run `go mod tidy` it'd also remove the indirects. But lmk if somehow they're needed I can add back.

Reply:
@wliao008 when I was testing your changes I had to do `go mod tidy` and it added those, also on a previous commit the build failed because of this.
This was the commit https://github.com/Shivansh-yadav13/SQLite-UI/commit/9e714dc85190895e54c231312d9d370816f99b07

so I had to make another commit doing `go mod tidy` https://github.com/Shivansh-yadav13/SQLite-UI/commit/4d455de1523497ac3528be491b6ff5bdbce08c8f

Reply:
yes, we are using 1.18 for the project